### PR TITLE
Adding ROSA 4.16

### PR DESCRIPTION
**Pull request by jtaleric**
Shifted cron 1hr to not collide with 4.15 jobs.

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-node-density-cni
+    workflow: rosa-aws-sts
+- as: conc-builds-3nodes
+  cron: 0 1 1,15 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:139]: not cron. Pre-submit

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-node-density-heavy
+    workflow: rosa-aws-sts
+- as: node-density-cni-24nodes
+  cron: 0 19 * * 5
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:123]: node-density-cni: not cron. Pre-submit

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      PROFILE_TYPE: reporting
+      REPLICAS: "120"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: rosa-aws-sts
+  timeout: 7h0m0s
+- as: control-plane-24nodes
+  cron: 0 12 * * 3
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:73]: 4.16 rosa AWS-OVN = Priority 1 => Weekly
Open slot: "0 1 2,9,16,23 * *"

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: rosa-aws-sts
+  timeout: 5h0m0s
+- as: data-path-9nodes
+  cron: 0 14 * * 3
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:90]: 4.16 rosa AWS-OVN = Priority 1 => Weekly
open slot: "0 1 1,8,15,22 * *"

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      REPLICAS: "9"
+    test:
+    - ref: openshift-qe-perfscale-aws-data-path-sg
+    - chain: openshift-qe-data-path-tests
+    workflow: rosa-aws-sts
+  timeout: 6h0m0s
+- as: node-density-heavy-24nodes
+  cron: 0 17 * * 4
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:107]: "0 18 5,12,19,26 * *"

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 6]
```diff
@@ -2249,6 +2249,578 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale
+    ci-operator.openshift.io/variant: rosa-4.16-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-cluster-density-v2-249nodes
```
Comment (afcollins) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2270]: Before we merge, let's add the slack reporter config to periodics file

**Comment by jtaleric** [comment]
@afcollins any other feedback or are we good to merge?

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 7]
```diff
@@ -2153,17 +2120,6 @@ periodics:
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-data-path-9nodes
-  reporter_config:
```
Comment (afcollins) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2156]: Can we keep these notifs?

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-heavy-24nodes-fips pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-heavy-24nodes-etcd-encrypt

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-conc-builds-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-120nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
```
time="2024-06-17 17:22:38" level=info msg="Evaluating alerts for prometheus: https://prometheus-k8s-openshift-monitoring.apps.ci-rosa-s-3wnh.3wbk.s1.devshift.org" file="alert_manager.go:125"
383
time="2024-06-17 17:22:39" level=warning msg="🚨 alert at 2024-06-17T17:11:54Z: '10 minutes avg. 99th etcd fsync latency on etcd-ip-10-0-3-194.us-west-2.compute.internal higher than 10ms. 0.01s'" file="alert_manager.go:205"
384
time="2024-06-17 17:22:40" level=error msg="alert at 2024-06-17T17:11:24Z: 'Critical prometheus alert. UpgradeConfigSyncFailureOver4HrSRE'" file="alert_manager.go:144"
385
time="2024-06-17 17:22:40" level=info msg="Indexing alerts" file="alert_manager.go:220"
386
time="2024-06-17 17:22:40" level=info msg="Indexing finished in 22ms: created=2" file="alert_manager.go:226"
387
time="2024-06-17 17:22:40" level=info msg="🔍 Endpoint: https://prometheus-k8s-openshift-monitoring.apps.ci-rosa-s-3wnh.3wbk.s1.devshift.org; profile: config/metrics-report.yml start: 2024-06-17T17:11:24Z end: 2024-06-17T17:17:38Z; job: node-density-heavy" file="prometheus.go:70"
388
time="2024-06-17 17:22:54" level=info msg="🔍 Endpoint: https://prometheus-k8s-openshift-monitoring.apps.ci-rosa-s-3wnh.3wbk.s1.devshift.org; profile: config/metrics.yml start: 2024-06-17T17:11:24Z end: 2024-06-17T17:17:38Z; job: node-density-heavy" file="prometheus.go:70"
389
time="2024-06-17 17:23:01" level=info msg="🔍 Endpoint: https://prometheus-k8s-openshift-monitoring.apps.ci-rosa-s-3wnh.3wbk.s1.devshift.org; profile: config/metrics-report.yml start: 2024-06-17T17:17:38Z end: 2024-06-17T17:22:38Z; job: garbage-collection" file="prometheus.go:70"
390
time="2024-06-17 17:23:15" level=info msg="🔍 Endpoint: https://prometheus-k8s-openshift-monitoring.apps.ci-rosa-s-3wnh.3wbk.s1.devshift.org; profile: config/metrics.yml start: 2024-06-17T17:17:38Z end: 2024-06-17T17:22:38Z; job: garbage-collection" file="prometheus.go:70"
391
time="2024-06-17 17:23:22" level=info msg="Indexing [2] documents from metric max-99thEtcdDiskBackendCommit-start" file="prometheus.go:230"
```

Getting another sample.

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-heavy-24nodes-etcd-encrypt

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-node-density-heavy-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

**Pull request closed without merging**